bigbluebutton-html5/imports/ui/components/timer/panel/styles.ts

```diff
@@ -191,52 +175,139 @@ const TimerTrackItem = styled.div`
   ${TimerRow}
 `;
 
-const TimerControls = styled.div`
-  display: flex;
-  width: 100%;
-  justify-content: center;
-  margin-top: 4rem;
-`;
-// @ts-ignore - JS code
-const TimerControlButton = styled(Button)`
-  width: 6rem;
-  margin: 0 1rem;
-`;
-
 const TimerInput = styled.input`
-  flex: 1;
-  border: 1px solid ${colorGrayLighter};
-  width: 50%;
+  background: none;
+  border: none;
+  border-bottom: 2px solid #e9e9ed;
+  color: inherit;
+  font-family: inherit;
+  font-size: inherit;
+  font-weight: ${textFontWeight};
+  font-variant-numeric: tabular-nums;
   text-align: center;
-  padding: .25rem;
-  border-radius: ${borderRadius};
-  background-clip: padding-box;
-  outline: none;
-
-  &::placeholder {
-    color: ${colorGray};
-    opacity: 1;
+  width: 2ch;
```

Comment:
What unit is this: `ch`? Can we use a common one here, like: %, vw, rem, em, ...?